game/world/objects/item.cpp

```diff
@@ -30,38 +29,38 @@ Item::Item(World &owner, size_t itemInstance, Type type)
 Item::Item(World &owner, Serialize &fin, Type type)
   :Vob(owner) {
   auto& h = hitem;
-  h.userPtr = this;
+  h = std::make_shared<phoenix::c_item>();
+  h->user_ptr = this;
 
   Tempest::Matrix4x4 mat;
 
   uint32_t instanceSymbol=0;
-  fin.read(instanceSymbol); h.instanceSymbol = instanceSymbol;
-  fin.read(h.id,h.name,h.nameID,h.hp,h.hp_max,h.mainflag);
-  fin.read(h.flags,h.weight,h.value,h.damageType,h.damageTotal,h.damage);
-  fin.read(h.wear,h.protection,h.nutrition,h.cond_atr,h.cond_value,h.change_atr,h.change_value,h.magic);
-  fin.read(h.on_equip,h.on_unequip,h.on_state);
-  fin.read(h.owner,h.ownerGuild,h.disguiseGuild,h.visual,h.visual_change);
-  fin.read(h.effect,h.visual_skin,h.scemeName,h.material);
-  fin.read(h.munition,h.spell,h.range,h.mag_circle);
-  fin.read(h.description,h.text,h.count);
-  fin.read(h.inv_zbias,h.inv_rotx,h.inv_roty,h.inv_rotz,h.inv_animate);
+  fin.read(instanceSymbol);
+
+  owner.script().initializeInstanceItem(h, instanceSymbol);
```

Comment:
As been mentioned before for npc's calling, during load, `initializeInstanceItem` is not correct